### PR TITLE
Clarified Running the Workshop

### PR DESCRIPTION
**Pull request by patrickstrick**
Added more details to the "Starting the Workshop for Participants"
section and cleaned up formatting (line breaks, etc) and spelling throughout doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +75,7 @@ Follow the directions in the [Running the Workshop](#running-the-workshop) secti
 The recommended way to deploy this workshop is directly from the RHPDS catalog as described above. If you'd like to deploy it manually, you can order the base OpenShift 4.2 Workshop and deploy the Getting Started workshop via the instructions below.
```
Comment (blues-man) [inline README.md]: Could please update your PR changing the `4.2` here with `4.6`? I left this old version for mistake, we could fix also this in this PR :)

**Review by blues-man — CHANGES_REQUESTED** [review]
LGTM, waiting for your changes to merge

File: README.md (hunk 1 above)
Comment (patrickstrick) [reply README.md]: @blues-man  Done. See 7c0d6b7b7560a4223697442e7a1369656ddb52f1

**Review by blues-man — APPROVED** [review]
+1

**Comment by blues-man** [comment]
Thanks @patrickstrick !

**Pull request merged**